core/meta/src/TStreamerElement.cxx

```diff
@@ -1878,19 +1878,17 @@ TClass *TStreamerSTL::GetClassPointer() const
    TClass *cl = TClass::GetClass(className, kTRUE, quiet);
 
    auto proxy = cl->GetCollectionProxy();
-   if (fNewClass && proxy->GetValueClass() == nullptr) {
+   if (!fNewClass && proxy && proxy->GetValueClass() == nullptr) {
```

Comment:
Just curious: in the previous version of this code, the if block was effectively never executed because first we test for `fNewClass` and later we skip if `fNewClass`? So this patch activates a previously dead code block?

Reply:
This is correct.